aten/src/ATen/native/verbose_wrapper.cpp

```diff
@@ -14,7 +14,8 @@ namespace verbose {
 
 TORCH_API int _mkl_set_verbose(int enable) {
```

Comment:
Do you mind changing return type to bool to indicate the intent here?
```suggestion
TORCH_API bool _mkl_set_verbose(int enable) {
```

Reply:
Yeah, sure. But will need to update other files as well in this invocation chain. I'll do it later.

Reply:
At the very least, add a comment, that it expects to return 0 on failure and 1 on success

Reply:
OK. I'll add the comment.